var/spack/repos/builtin/packages/libxml2/package.py

```diff
@@ -162,3 +142,61 @@ def test(self):
 
         # Perform some cleanup
         fs.force_remove(test_filename)
+
+
+class RunAfter(object):
+    @run_after("install")
+    @on_package_attributes(run_tests=True)
+    def import_module_test(self):
+        if "+python" in self.spec:
+            with working_dir("spack-test", create=True):
+                python("-c", "import libxml2")
+
+
+class AutotoolsBuilder(AutotoolsBuilder, RunAfter):
+    def configure_args(self):
+        spec = self.spec
+
+        args = [
+            "--with-lzma={0}".format(spec["xz"].prefix),
+            "--with-iconv={0}".format(spec["iconv"].prefix),
+        ]
+
+        if "+python" in spec:
+            args.extend(
+                [
+                    "--with-python={0}".format(spec["python"].home),
+                    "--with-python-install-dir={0}".format(python_platlib),
+                ]
+            )
+        else:
+            args.append("--without-python")
+
+        return args
+
+
+class NMakeBuilder(NMakeBuilder, RunAfter):
+    phases = ("configure", "build", "install")
+
+    @property
+    def makefile_name(self):
+        return "Makefile.msvc"
+
+    @property
+    def build_directory(self):
+        return os.path.join(super().build_directory, "win32")
+
+    def configure(self, pkg, spec, prefix):
+        with working_dir(os.path.join(self.stage.source_path, "win32")):
```

Comment:
Is this `build_directory`?

Reply:
Seems to be and if it isn't it should be, amended and pushed.